### PR TITLE
Some good practices for l10n.

### PR DESCRIPTION
**Pull request by davedash**

### CODE REVIEW

File: l10n.rst [hunk 1]
```diff
@@ -119,3 +119,60 @@ In your projects ``settings.py`` add the new locale to ``KNOWN_LANGUAGES``.
...
+    <h1>_('Hello')</h1>
+
+    {% trans about_url='http://about.me/lionel.richie' %}
+      <p>
+        Is it <a href="{{ about_url }}">me</a> you're looking for?
+      </p>
+      <p>
+        I can see it in your eyes.
+      </p>
+    {% endtrans %}
```
Comment (Osmose) [inline l10n.rst:170]: Putting the `<p>` tags in the trans block is a little iffy to me; couldn't this be better accomplished by a trans block for the first paragraph and ugettext for the second?

Of course, there could be localization issues relating to the paragraphs here, so I could be missing that aspect.

**Comment by Osmose** [comment]
r+, the trans block stuff is more a curiosity than a real problem, and it conveys the point of moving the h1 out and using trans block vars quite nicely.

File: l10n.rst (hunk 1 above)
Comment (fwenzel) [inline l10n.rst:170]: Yeah I wouldn't put tags inside a gettext block unless they are likely to be moved around by localizers (like links on idividual words pretty much always are).

Comment (davedash) [inline l10n.rst:170]: Typically there will be multiple paragraphs, which is why the trans block is awesome.

Comment (davedash) [inline l10n.rst:170]: Think about an about page, or a several paragraph book.

Comment (davedash) [inline l10n.rst:170]: Fuu... I just saw this.  See my comments, I can perhaps adjust the comments
to explain it.

-d

**Pull request merged**